freqtrade/optimize/optimize_reports.py

```diff
@@ -509,12 +516,22 @@ def generate_backtest_stats(btdata: Dict[str, DataFrame],
     :param max_date: Backtest end date
     :return: Dictionary containing results per strategy and a strategy summary.
     """
-    result: Dict[str, Any] = {'strategy': {}}
+    result: Dict[str, Any] = OrderedDict([
+        ('metadata', {}),
+        ('strategy', {}),
+        ('strategy_comparison', []),
+    ])
     market_change = calculate_market_change(btdata, 'close')
+    metadata = result['metadata']
     pairlist = list(btdata.keys())
     for strategy, content in all_results.items():
         strat_stats = generate_strategy_stats(pairlist, strategy, content,
                                               min_date, max_date, market_change=market_change)
+        # Warning: due to implementation details of get_backtest_metadata() function, characters
+        # { and } appearing in strings will break stuff.
```

Comment:
It's unclear what you mean with this warning (it refers to a method that does not exist).
Is this because of `get_backtest_metadata_filename` - or `strategy_run_id` (or something else)

Also - can you make an example what would need to be the input for this to break?

Reply:
Leftover from previous hacky parsing of json - removing 👍🏻